data/topic-schema/src/main/resources/net/corda/schema/VirtualNode.yaml

```diff
@@ -76,3 +76,17 @@ topics:
       min.compaction.lag.ms: 60000
       max.compaction.lag.ms: 604800000
       min.cleanable.dirty.ratio: 0.5
+  VirtualNodeOperationStatusTopic:
+    name: virtual.node.operation.status
+    consumers:
+      - db
+    producers:
+      - rpc
+    config:
+  VirtualNodeAsyncRequestTopic:
+    name: virtual.node.async.request
+    consumers:
+      - db
+    producers:
+      - rpc
+    config:
```

Comment:
I'm not sure if these are the correct consumers/producers for these changes. Will follow up.